tap_helpscout/schemas/happiness_ratings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "thread_id": {
+      "type": ["null", "integer"]
+    },
+    "conversation_id": {
+      "type": ["null", "integer"]
+    },
+    "thread_created_at": {
+      "type": ["null", "string"],
+      "format": "date-time"
+    },
+    "rating_id": {
+      "type": ["null", "integer"]
+    },
+    "rating": {
+      "type": ["null", "string"]
```

Comment:
See comment on transform.py re product input on whether or not we should be generating a field that's not returned by the API.

Reply:
Product confirmed this should be excluded.